### PR TITLE
build: convert vg manager clusterrole into role as per requirements 

### PR DESCRIPTION
**Pull request by iamniting**
Signed-off-by: Nitin Goyal <nigoyal@redhat.com>

### CODE REVIEW

**Review by leelavg — APPROVED** [review]
/lgtm

File: config/rbac/vg_manager_cluster_role_binding.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
```
Comment (nbalacha) [inline config/rbac/vg_manager_cluster_role_binding.yaml:1]: Nit. Please rename the file to vg_manager_clusterrole_binding.yaml to match the other files.

File: config/rbac/vg_manager_cluster_role.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```
Comment (nbalacha) [inline config/rbac/vg_manager_cluster_role.yaml:3]: Nit. Please rename the file to vg_manager_clusterrole.yaml

**Review by nbalacha — CHANGES_REQUESTED** [review]

File: config/rbac/vg_manager_cluster_role_binding.yaml (hunk 1 above)
Comment (iamniting) [reply config/rbac/vg_manager_cluster_role_binding.yaml]: done

File: config/rbac/vg_manager_cluster_role.yaml (hunk 2 above)
Comment (iamniting) [reply config/rbac/vg_manager_cluster_role.yaml]: done

File: config/rbac/kustomization.yaml [hunk 3]
```diff
@@ -31,4 +31,6 @@ resources:
 - auth_proxy_client_clusterrole.yaml
 - vg_manager_role.yaml
 - vg_manager_role_binding.yaml
+- vg_manager_cluster_role.yaml
```
Comment (nbalacha) [inline config/rbac/kustomization.yaml:34]: These would need to be updated as well.

Comment (iamniting) [reply config/rbac/kustomization.yaml]: yes correct

**Review by nbalacha — APPROVED** [review]

**Pull request merged**